src/order/basic.lean

```diff
@@ -770,6 +770,22 @@ instance : densely_ordered punit := ⟨λ _ _, false.elim⟩
 
 end punit
 
+section prop
```

Comment:
```suggestion
section Prop
```

Reply:
This seems to be a syntax error.

Reply:
Oh well :(

Reply:
You can probably use french quotes, but given it's just a section and not a namespace it doesn't matter anyway.